### ISSUE TITLE
Question: Is the route limited to the routes in server.fs

### ISSUE THREAD

**Issue by mastoj** [dbrattli/Felizia#11]
Readin through the code in mode detail and found this: https://github.com/dbrattli/Felizia/blob/master/Felizia/server/Server.fs#L119-L128

Does this mean the level of nesting is limited to for levels deep at the moment? Not that I think it will be an issue for me, just curious.

**Comment by dbrattli** [comment]
Yes, that is a correct observation. I'm struggling a bit with setting up the routing so hard coded it for now. Will fix it some other time when I (or someone else) figure out a better way to handle this.
